frontend/src/app/organization/organization-details/organization-details.component.html

```diff
@@ -1,4 +1,8 @@
 <div class="container">
+  <div class="card bg-light mb-3 text-white">
```

Comment:
The whole point of this change was to introduce solution, that will work without any changes when backend API will be developed. You can use `loadDefaultImage` the way how @juliaanholcer developed it for offerrs or suggest your own solution, but it should be already here, when backend will arrive.

Reply:
loadDefaultImage is only for offer, should I make other function for organization?